pkgs/shells/bash/default.nix

```diff
@@ -113,6 +113,6 @@ stdenv.mkDerivation rec {
   };
 
   passthru = {
-    shellPath = "/bin/bash";
+    shellPath = "${bash}/bin/bash";
```

Comment:
Oops, this might not make sense for `bash` itself ;)  I only know `passthru` not exactly what it is used for, though.